### DISCUSSION TITLE
Flickering between frames

### DISCUSSION THREAD

**Discussion by A-Guidry** [neverhood311/Stop-motion-OBJ#175] [Q&A]
Hi, first off, thank you for creating this plugin, it is awesome and needed in the community. 

My question is model noise or artifacts when playing back. In the video attached, this is most notable in the pants. Are there any tricks based on the import setting or rendering setting to help reduce the noise/artifacting when exporting?

Thank you for your help and response

https://user-images.githubusercontent.com/17969493/225800543-446ec604-969c-49dd-80da-9604bf83a93f.mp4

https://user-images.githubusercontent.com/17969493/225800625-f96dbfef-af7b-46fc-b915-e47c8cb7fd01.mp4

**Comment by neverhood311** [comment]
Hi Anthony, I'm glad you like Stop Motion OBJ!

> Are there any tricks based on the import setting or rendering setting to help reduce the noise/artifacting when exporting?

The addon imports meshes as-is and there are no options for processing the meshes. In your case (I'm assuming this is a 4D photogrammetry scan), there are probably tools for mesh sequence cleanup. Unfortunately, that's outside of the scope of SMO and also outside of my area of expertise.

If you want to do cleanup entirely within Blender and SMO, your best option would be to import your mesh sequence and manually edit your meshes one-by-one (tedious, I know). If you cleaned up your sequence and want to export it back out to .OBJ files, you could try out the new `Export`* feature (currently in [v2.2.0.beta.0](https://github.com/neverhood311/Stop-motion-OBJ/releases/tag/v2.2.0.beta.0)), which was commissioned by Lee Perry Smith.

*Disclaimer: the Export feature is still in Beta, so it couldn't hurt to back up the original files of your mesh sequence before testing it out.
